Server/Game/routegamehelper.py

```diff
@@ -293,7 +305,18 @@ def queryGamesUser(data):
 					game_response["Enemy_Perks"].append({"Name": opp_game_user.perk_3.name, "Tier":3})
 
 				# Store all of the Actions from History for the game
-				action_history = Action_History.objects.filter(game=game).order_by("order")
+				turn_start = game.game_round - 1
+				turn_end = game.game_round
+
+				if filters and 'Num_Prev_Turns' in filters:
+					turn_start = game.game_round - (int(filters['Num_Prev_Turns']) - 1)
+
+				if turn_start < 0:
+					turn_start = 0
+
+				logging.info("Fetching Action History for turns {0}-{1}".format(turn_start, turn_end))
```

Comment:
Should probably become debug level, or stay info, but include gameID, user, or other info?